types/chai/index.d.ts

```diff
@@ -34,6 +36,10 @@ declare namespace Chai {
         (target: any, message?: string): Assertion;
     }
 
+    export type Operator = string; // "==" | "===" | ">" | ">=" | "<" | "<=" | "!=" | "!==";
```

Comment:
Can't 😦 see above conversation. This and the `keyof` types are blocked on updating libraries that depend on chai to also require TS 2.1. I'll do that in a subsequent PR.